projects/js-packages/api/index.jsx

```diff
@@ -160,6 +160,11 @@ function JetpackRestApiClient( root, nonce ) {
 				.then( checkStatus )
 				.then( parseJsonResponse ),
 
+		hasSeenWCConnectionModal: () =>
```

Comment:
This sets the value, right? The naming of it seems a little confusing to me in that it sounds like it checks if it's been seen, rather than sets a value indicating that it's been seen.
eg. The user _has_ seen the modal